teamtwo/accounts/templates/registration/password_reset_done.html

```diff
@@ -0,0 +1,6 @@
+{% block title %}Email Sent{% endblock %}
+{% block content %}
+<h1>Check your inbox.</h1>
+<p>We've emailed you instructions for setting your password. You should receive the email shortly!</p>
```

Comment:
I think it can be rewrite as:
"The instruction has been sent to your email address. Please check your inbox."